browser/pidregistry.go

```diff
@@ -0,0 +1,28 @@
+package browser
+
+import "sync"
+
+// pidRegistry keeps track of the launched browser process IDs.
+type pidRegistry struct {
+	mu  sync.RWMutex
+	ids []int
+}
+
+// registerPid registers the launched browser process ID.
+func (r *pidRegistry) registerPid(pid int) {
+	r.mu.Lock()
+	defer r.mu.Unlock()
+
+	r.ids = append(r.ids, pid)
+}
+
+// PIDs returns the launched browser process IDs.
```

Comment:
s/PIDS/Pids ?

Reply:
Yeah, I've noticed it now 😅 Thanks!

Reply:
Resolved in https://github.com/grafana/xk6-browser/pull/779/commits/cedf4200402d79a514ca3040bea6241e93bd97d0.